packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,272 @@
 # @fluentui/react-persona Spec
 
+Convergence epic issue: #24213
+
 ## Background
 
-_Description and use cases of this component_
+A Persona is a visual representation that showcases an Avatar, Icon, Image, or Badge.
+
+Persona is used in PeoplePicker, Team's left rail and menus, chiclets, and Card.
```

Comment:
I think it is worth calling out that this is a composite component but is not a card in itself - it won't be providing a visual board or popup when the user hovers over an avatar.